### ISSUE TITLE
Services API "Custom Issue Tracker"

### ISSUE THREAD

**Issue by philippe-granet** [gitlab4j/gitlab4j-api#404]
Can you please implement "Custom Issue Tracker" service management ?
https://docs.gitlab.com/ee/api/services.html#custom-issue-tracker

**Comment by gmessner** [comment]
@philippe-granet 
Any possibility that you could implement this?  All contributions are welcome here.

**Comment by gmessner** [comment]
@philippe-granet 
Actually, after taking a better look at this, this service is a bit different and similar to several others.  I will go ahead and implement it and schedule it for the next release which should be Friday 7/12/2019.

**Comment by gmessner** [comment]
@philippe-granet 
Had some time this evening so I went ahead and implemented the Custom Issue Tracker and Bugzilla services.  Please verify and close.

**Comment by philippe-granet** [comment]
@gmessner Hi, thank you for your response! it seems to work.
 But I think CustomIssueTrackerService should not extends NotificationService.
CustomIssueTrackerService only use `title` attribute of NotificationService class, all others attributes aren't used.
There are severals type of Services :  NotificationService, IssueService, ... But only a few of these services share same attributes

**Comment by gmessner** [comment]
@philippe-granet 
Use Postman or something like it and look at the JSON returned by the call to GET Custom Issue Tracker (https://docs.gitlab.com/ee/api/services.html#get-custom-issue-tracker-service-settings) and every other service currently implemented,  you will see that they all share the fields that are in the ```NotificationService```.  This is why all the current services are subclasses of ```NotificationService``` 

The documentation for the services is lacking any examples of the returned results, so I always use Postman now to verify what is returned.  I have also found out that the GitLab API documentation is often not correct, which is why I will verify things with Postman.  I am working on submitting a bunch of fixes to the GitLab API docs.

When verifying the above I did find one field in ```NotificationService``` that should not be there, ```commit_events```, this should be moved to ```JiraService```, I have created an Issue for this bug (#405)
